mergesort.java

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
Please move this to the correct folder. 
https://github.com/Kavya-24/Snippets/tree/master/Algorithms/Sorting%20Algorithms